File: LDC.Atlas.Services.Trading/Entities/Costmatrix.cs
using System.Collections.Generic;

namespace LDC.Atlas.Services.Trading.Entities
{
    public class CostMatrix
    {
        public long CostMatrixId { get; set; }

        public string Name { get; set; }

        public string Description { get; set; }

        public IEnumerable<CostMatrixLine> CostMatrixLines { get; set; }
    }
}

File: LDC.Atlas.WebApp/Client/app/shared/services/trading/dtos/costmatrix.ts
import { Tag } from '../../../entities/tag.entity';
import { CostmatrixLine } from './costmatrixLine';

export class Costmatrix {
    costMatrixId: number;
    name: string;
    description: string;
    costMatrixLines: CostmatrixLine[];
    createdBy: string;
    createdDateTime: Date;
    tags: Tag[];
    bestMatch: number;
    tagsFormatted: string[];
}
